pyproject.toml

```diff
@@ -73,17 +73,11 @@ notebooks = [
   "ipysheet"
 ]
 
-[tools.setuptools]
-include-package-data = true
-
 [tool.setuptools.packages.find]
 where = ["."]
 include = ["pgfinder"]
 exclude = ["tests"]
 
-[tools.setuptools.package-data]
-pgfinder = ["*.yaml"]
```

Comment:
As you want to add `.csv` files to the package data I think you need to reinstate this configuration block and add `.csv` to the list...

```toml
pgfinder = ["*.yaml", "*.csv"]
```